models/DataObject/ClassDefinition/Data/Objectbricks.php

```diff
@@ -740,39 +740,46 @@ public function checkValidity($data, $omitMandatoryCheck = false)
             if ($data instanceof DataObject\Objectbrick) {
                 $validationExceptions = [];
 
-                $items = $data->getItems();
```

Comment:
@weisswurstkanone I'm wondering why the empty brick isn't included in `getItems()` shouldn't it be there as well?

Reply:
@brusch at that point of time items is not set for the newly created brick